src/util/Ip.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ByteArray } from "@project-chip/matter.js";
+
+export function isIPv4(ip: string) {
+    return ip.includes(".")
```

Comment:
nice pragmatic solution preventing a regex :-)

Reply:
Quick and dirty :-)
This doesn't ensure that the string is a well-formed IPv4 address but good enough when the IP comes from another API.